### PR TITLE
version compare keyword

### PR DESCRIPTION
**Pull request by lugi0**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -29,6 +29,7 @@
         "requests",
         "robotframework-requests",
         "escapism",
+        "semver>=2,<3"
```
Comment (vasukulkarni) [inline setup.py:32]: is there a reason for version b/w 2 and 3?

File: setup.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
         "robotframework-requests",
         "robotframework-seleniumlibrary",
         "robotframework-jupyterlibrary>=0.3.1",
-        "robotframework-OpenShiftCLI",
+        "robotframework-OpenShiftCLI>=1.0.1",
```
Comment (vasukulkarni) [inline setup.py:23]: can we pin to 1.0.1 ?  use `==`

File: setup.py (hunk 1 above)
Comment (lugi0) [reply setup.py:32]: v3 has breaking changes but I think it's still not released, my pip install got me version 2 by default

File: setup.py (hunk 2 above)
Comment (lugi0) [reply setup.py:23]: sure

**Review by vasukulkarni — APPROVED** [review]

**Comment by vasukulkarni** [comment]
@jgarciao please cherrypick and let me know if it worked

**Comment by jgarciao** [comment]
I integrated the commits in #138

**Pull request merged**